### PR TITLE
Incident Extension Rework

### PR DESCRIPTION
**Pull request by dc3-tsd**
Reworking the Core Incident Extension 1.0 into several separate extensions and documenting all of these using adoc.  This branch is expected to undergo changes based on reviews and feedback by the TC as well as interested contributors and reviewers on GitHub.  Many of the examples are currently incorrect and do not match the JSON schemas as these components continue to be reworked.

### CODE REVIEW

File: extension-definition-specifications/incident-core/Incident Extension Suite.adoc [hunk 1]
```diff
@@ -891,6 +896,10 @@ This list *MUST* not contain cycles.
 
 // tag::task-relationships[]
 
+When creating sequences of [stixtype]#<<task,tasks>># these *SHOULD NOT* be shared using relationship objects.
```
Comment (adulau) [inline extension-definition-specifications/incident-core/Incident Extension Suite.adoc:899]: Just to be sure it was intended, it leaves the option open to use relationships type as a SHOULD specified.

**Comment by adulau** [comment]
Maybe an additional minor point, concerning `5.9. Task Outcome Enumeration`, should we add `discarded` ? which is task which intended to be discarded during an investigation.

File: extension-definition-specifications/incident-core/Incident Extension Suite.adoc (hunk 1 above)
Comment (adulau) [reply extension-definition-specifications/incident-core/Incident Extension Suite.adoc:899]: During the last call, we need to check if the extension best practices allow to enforce a direction concerning relationships.

**Comment by ejratl** [comment]
Thank you for this incredible work, @dc3-tsd

**Pull request merged**